tasks/hyrax-metadata-updates/tests/test-integration-index.js

```diff
@@ -80,25 +81,31 @@ test.before(async () => {
   }).promise();
 });
 
-test.beforeEach(() => {
+const setupNock = (params) => {
+  const cmrParams = merge({
```

Comment:
You can probably get rid of the merge dependency and just use an object spread operator here. 
something like  
```
  const cmrParams = {
    ...params,
    page_size: '50',
    page_num: '1',
    provider_short_name: 'GES_DISC',
  };
```
And then you can remove L11 also

Reply:
I'll update this in the Cumulus branch